disk/tests/test_filter.py

```diff
@@ -34,12 +34,12 @@ def test_bad_config_string_regex():
     }
     c = Disk('disk', None, {}, [instance])
 
-    assert c._file_system_whitelist.pattern == 'test'
-    assert c._file_system_blacklist.pattern == 'test|iso9660$'
-    assert c._device_whitelist.pattern == 'test'
-    assert c._device_blacklist.pattern == 'test'
-    assert c._mount_point_whitelist.pattern == 'test'
-    assert c._mount_point_blacklist.pattern == 'test'
+    assert c._file_system_whitelist == re.compile('test', re.I)
+    assert c._file_system_blacklist == re.compile('test|iso9660$', re.I)
+    assert c._device_whitelist == re.compile('test', IGNORE_CASE)
+    assert c._device_blacklist == re.compile('test', IGNORE_CASE)
+    assert c._mount_point_whitelist == re.compile('test', IGNORE_CASE)
+    assert c._mount_point_blacklist == re.compile('test', IGNORE_CASE)
```

Comment:
Why both `re.I` and `IGNORE_CASE` are used ?

I think it may help to have an explanation comment for this line: https://github.com/DataDog/integrations-core/blob/0ab20abae10e812a6723e41861ac9ac7e3ba73ad/disk/datadog_checks/disk/disk.py#L30

Reply:
Added a comment 🙂